### PR TITLE
Drop python3.7, use newest cookiecutter

### PR DESCRIPTION
**Pull request by Zeitsperre**

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -11,17 +11,18 @@ In a situation where Raven configuration files (`.rv*` files) exist already, Rav
...
 .. code-block:: python
 
-   from ravenpy.models import Raven
-   model = ravenpy.models.Raven(workdir="/tmp/testrun")
-   model.configure([<list of .rv* files>])
-   model([<list of forcing files>])
+    from ravenpy.models import Raven
+
+    model = ravenpy.models.Raven(workdir="/tmp/testrun")
+    model.configure(["rvfile1.rv", "rvfile2.rv", "rvfile3.rv", ...])
```
Comment (huard) [inline docs/usage.rst:17]:
```suggestion
    model.configure(["conf.rvi", "conf.rvt", "conf.rvh", "conf.rvc", "conf.rvp"])
```

File: docs/usage.rst [hunk 2]
```diff
@@ -11,17 +11,18 @@ In a situation where Raven configuration files (`.rv*` files) exist already, Rav
...
+    model(["forcingfile1", "forcingfile2", "forcingfile3", ...])
```
Comment (huard) [inline docs/usage.rst:18]:
```suggestion
    model(["tas.nc", "pr.nc"])
```

File: docs/usage.rst [hunk 3]
```diff
@@ -32,10 +33,11 @@ and then pass `start_date` and `end_date` as arguments to the `model` call:
...
 .. code-block:: python
 
-  import datetime as dt
-  model = ravenpy.models.Raven()
-  model.configure([<list of .rv* file templates>])
-  model(start_date=dt.datetime(2020, 1, 1), end_date=dt.datetime(2020, 2, 1))
+    import datetime as dt
+
+    model = ravenpy.models.Raven()
+    model.configure(["rvfile1.rv", "rvfile2.rv", "rvfile3.rv", ...])
```
Comment (huard) [inline docs/usage.rst:39]:
```suggestion
    model.configure(["conf.rvi", "conf.rvt", "conf.rvh", "conf.rvc", "conf.rvp"])
```

File: docs/usage.rst [hunk 4]
```diff
@@ -53,24 +55,26 @@ For each one of these, `.rv` files are provided that reproduce almost perfectly
...
-         elevation=843.0,
-         latitude=54.4848,
-         longitude=-123.3659
-         )
-   model.q_sim.plot()
+    from ravenpy.models import GR4JCN
+
+    model = GR4JCN()
+    model(
+        ts=["forcingfile1", "forcingfile2", "forcingfile3", ...],
```
Comment (huard) [inline docs/usage.rst:62]:
```suggestion
        ts=["meteo.nc"],
```

**Review by huard — APPROVED** [review]

**Comment by Zeitsperre** [comment]
Tests are currently failing due to a service outage at our internal servers. Will rerun tests when servers are back online.

**Comment by Zeitsperre** [comment]
There seems to be even more issues with the newest release of netcdf4-python (see: https://github.com/Unidata/netcdf4-python/issues/1192). Will attempt to pin at 1.6.0, otherwise below. There's an issue with the wheel/binary-building process.

**Comment by huard** [comment]
Ok, no need to be on the bleeding edge here.

**Pull request merged**